policy/policy.go

```diff
@@ -114,14 +125,24 @@ func (c *ARC[T]) Clear() {
 }
 
 func (c *ARC[T]) Values() iter.Seq[T] {
+	// ping pong between recent/frequent.
```

Comment:
realized this is a better representation of hot->cold with ARC. Before it just considered frequent to trump recent, which is antithetical.

Reply:
seems good! but doesn't this break the "Hottest to coldest" guarantee that is documented above? are callers not relying on the previous semantics?

Reply:
It improves it! :D it takes the hottest from recent and returns, then hottest from frequent and returns, and then next hottest from recent etc.

This is only used in our file cache when writing the manifest, so low risk.